include/affinity.imp.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#ifdef _WIN32
+
+#include "affinity.win.h"
```

Comment:
Do we want to follow this approach in terms of multiplatform support?

I'd personally at least not want this to be directly in `include/`

Reply:
Agreed, especially since we might be getting more of these soon (cc @BlackMark29A).

Reply:
How about this folder structure:
`include/affinity.h`
`include/platform_specific/affinity_unix.h`
`include/platform_specific/affinity_windows.h`

Or do we want to have the platform specific code somewhere else entirely, not even inside `include`?